libs/shared/src/consts/providers/provider.interface.ts

```diff
@@ -4,223 +4,268 @@ export interface IProviderConfig {
   {
-    key: 'from',
+    key: 'user',
+    value: 'User',
+    type: 'string',
+  },
+  {
+    key: 'password',
+    value: 'Password',
     type: 'string',
   },
   {
     key: 'host',
+    value: 'Host',
     type: 'string',
   },
   {
     key: 'port',
+    value: 'Port',
     type: 'number',
   },
   {
     key: 'secure',
+    value: 'Secure',
     type: 'boolean',
   },
   {
-    key: 'user',
-    type: 'string',
-  },
-  {
-    key: 'password',
+    key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const postmarkConfig = [
+export const postmarkConfig: IConfigCredentials[] = [
   {
     key: 'apiKey',
+    value: 'API Key',
     type: 'string',
   },
   {
     key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const sendgridConfig = [
+export const sendgridConfig: IConfigCredentials[] = [
   {
     key: 'apiKey',
+    value: 'API Key',
     type: 'string',
   },
   {
     key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const sendinblueConfig = [
+export const sendinblueConfig: IConfigCredentials[] = [
   {
     key: 'apiKey',
+    value: 'API Key',
     type: 'string',
   },
 ];
 
-export const sesConfig = [
+export const sesConfig: IConfigCredentials[] = [
   {
-    key: 'region',
+    key: 'accessKeyId',
+    value: 'Access key ID',
     type: 'string',
   },
   {
-    key: 'accessKeyId',
+    key: 'secretAccessKey',
+    value: 'Secret access key',
     type: 'string',
   },
   {
-    key: 'secretAccessKey',
+    key: 'region',
+    value: 'Region',
     type: 'string',
   },
   {
     key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const plivoConfig = [
+export const plivoConfig: IConfigCredentials[] = [
   {
     key: 'accountSid',
+    value: 'Account SID',
     type: 'string',
   },
   {
     key: 'authToken',
+    value: 'Auth token',
     type: 'string',
   },
   {
     key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const sms77Config = [
+export const sms77Config: IConfigCredentials[] = [
   {
     key: 'apiKey',
+    value: 'API Key',
     type: 'string',
   },
   {
     key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const snsConfig = [
+export const snsConfig: IConfigCredentials[] = [
   {
     key: 'accessKeyId',
+    value: 'Access key ID',
     type: 'string',
   },
   {
     key: 'secretAccessKey',
+    value: 'Secret access key',
     type: 'string',
   },
 ];
 
-export const telnyxConfig = [
+export const telnyxConfig: IConfigCredentials[] = [
   {
     key: 'apiKey',
+    value: 'API Key',
     type: 'string',
   },
   {
-    key: 'from',
+    key: 'messageProfileId',
+    value: 'Message profile ID',
     type: 'string',
   },
   {
-    key: 'messageProfileId',
+    key: 'from',
+    value: 'From',
     type: 'string',
   },
 ];
 
-export const twilioConfig = [
+export const twilioConfig: IConfigCredentials[] = [
   {
     key: 'accountSid',
+    value: 'Account SID',
```

Comment:
Maybe a better name for this: displayName or title as value might be the actual value the user enters in the field and this might confuse a bit

Reply:
I can see the possibility of confusion I will update it for more appropriate name.